src/main/java/uk/gov/hmcts/divorce/document/content/DraftPetitionTemplateContent.java

```diff
@@ -101,9 +101,19 @@
         templateData.put(HAS_FINANCIAL_ORDERS_FOR_CHILD, hasFinancialOrdersForChild);
         templateData.put(FINANCIAL_ORDER_CHILD, CHILDREN_OF_THE_APPLICANT_AND_THE_RESPONDENT);
 
-        String respondentPostalAddress = null == caseData.getRespondentHomeAddress()
-            ? caseData.getDerivedRespondentSolicitorAddr()
-            : caseData.getRespondentHomeAddress();
+        String respondentPostalAddress;
+        if (caseData.getRespondentHomeAddress() == null) {
+            respondentPostalAddress = caseData.getDerivedRespondentSolicitorAddr();
+        } else {
+            respondentPostalAddress =
+                caseData.getRespondentHomeAddress().getAddressLine1() + "\n"
+                + caseData.getRespondentHomeAddress().getAddressLine2() + "\n"
+                + caseData.getRespondentHomeAddress().getAddressLine3() + "\n"
+                + caseData.getRespondentHomeAddress().getPostTown() + "\n"
+                + caseData.getRespondentHomeAddress().getCounty() + "\n"
+                + caseData.getRespondentHomeAddress().getPostCode() + "\n"
+                + caseData.getRespondentHomeAddress().getCountry() + "\n";
```

Comment:
can use StringJoiner here

Reply:
Used streams instead of StringJoiner in the end as StringJoiner didn't have a way of dealing with null values in an elegant way.